source/blog/2017-11-15-migrating-kubernetes-on-fedora-atomic-host-27.html.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Migrating Kubernetes on Fedora Atomic Host 27
+author: jbrooks
+date: 2017-11-15 00:00:00 UTC
+tags: atomic, kubernetes, fedora, system containers, rpm-ostree
+comments: true
+published: true
+---
+
+Starting with version 27 of Fedora Atomic Host, the rpms for Kubernetes, Flannel and Etcd are no longer baked into the host's image, but are installable instead either as [system containers](http://www.projectatomic.io/blog/2016/09/intro-to-system-containers/) or via [package layering](https://rpm-ostree.readthedocs.io/en/latest/manual/administrator-handbook/#hybrid-imagepackaging-via-package-layering).
```

Comment:
would prefer  **Starting with Fedora 27 Atomic Host**.. also weird. for some reason I thought we left `etcd` in.